packages-internal/test-utils/src/describeSkipIf.tsx

```diff
@@ -0,0 +1,9 @@
+// Shim for vitest describe.skipIf to be able to run mocha and vitest side-by-side
+// TODO: Remove after migration to vitest is complete
+const describeSkipIf: (condition: boolean) => Mocha.PendingSuiteFunction =
```

Comment:
I hope it's just temporary, as the name is terrible :)

Reply:
Yes, we'll turn this into `describe.skipIf` once we're away from mocha. There is no way to monkey patch in mocha, they don't expose these in the setup files